port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example

```diff
@@ -0,0 +1,5 @@
+OCEAN__PORT__CLIENT_ID="<port-client-id>"
+OCEAN__PORT__CLIENT_SECRET="<port-client-secret>"
+OCEAN__EVENT_LISTENER={"type": "POLLING"}
+OCEAN__INTEGRATION__IDENTIFIER="my-{{ }}-integration"
+OCEAN__INTEGRATION__TYPE="aws"
```

Comment:
Maybe change from aws?

Reply:
That's the thing though, we need it to be one of the default ones for it to actually work on the initial try :(, 
the `{{ }}` part is a bug though!

Reply:
I dont agree with this one
Lets find a better solution